pkg/util/framer/framer.go

```diff
@@ -21,6 +21,7 @@ import (
 	"encoding/binary"
 	"encoding/json"
 	"io"
+	"io/ioutil"
```

Comment:
@smarterclayton @krousey - I'm not entirely sure this PR is even correct. But just throwing it to start discussing about it.